liquibase-core/src/main/java/liquibase/changelog/ChangeSet.java

```diff
@@ -955,14 +955,26 @@ public boolean isAlwaysRun() {
         return alwaysRun;
     }
 
+    public void setAlwaysRun(boolean alwaysRun) {
```

Comment:
Why did you make these settable @tms-91 ? 

We've been trying to keep ChangeSet immutable vs. how it was originally constructed to be safe. But do you have a use case where you need to change them?

Reply:
Thanks! Looks good